src/Features/Core/Portable/ExternalAccess/VSTypeScript/VSTypeScriptNavigateToSearchService.cs

```diff
@@ -60,14 +55,9 @@ internal class VSTypeScriptNavigateToSearchService : INavigateToSearchService
             ImmutableArray<Document> priorityDocuments,
             string searchPattern,
             IImmutableSet<string> kinds,
-            bool isFullyLoaded,
             Func<INavigateToSearchResult, Task> onResultFound,
             CancellationToken cancellationToken)
         {
-            // We only support searching when the project is fully loaded.
-            if (!isFullyLoaded)
-                return;
```

Comment:
SearchProject and SearchDocument are only ever called now when the solution is loaded, so this value is not passed along, and so TS and F# don't even need to care about the 'false' case.